src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -148,7 +148,8 @@ def collect_data(geop_result, geojson):
     z['UrbAreaTotal'] = sum(z['Area'][NRur:])
     z['PhosConc'] = phosphorus_conc(z['SedPhos'])
 
-    z['NormalSys'] = normal_sys(z['Area'])
+    normal_sys_estimate = int(num_normal_sys(z['Area']))
```

Comment:
Done! Also updated the docstring for that method to explain why it was returning an array of ints.